src/BookingForm.js

```diff
@@ -0,0 +1,98 @@
+import React, { useState } from "react";
+import "./BookingForm.css";
+
+const BookingForm = props => {
+  const [newBooking, setNewBooking] = useState({
+    id: null,
+    firstName: "",
+    surname: "",
+    email: "",
+    roomId: null,
+    checkInDate: "",
+    checkOutDate: ""
+  });
+  const handleChange = e => {
+    const updatedBooking = { ...newBooking, [e.target.id]: e.target.value };
```

Comment:
Excellent use of `...` to build the new object

Reply:
I also like the fact that you've been able to reuse the same `handleChange` function in order to create the `updatedBooking` object